src/wp-includes/class-walker-nav-menu.php

```diff
@@ -171,6 +171,7 @@ public function start_el( &$output, $item, $depth = 0, $args = array(), $id = 0
 		$atts['target'] = ! empty( $item->target )     ? $item->target     : '';
 		$atts['rel']    = ! empty( $item->xfn )        ? $item->xfn        : '';
 		$atts['href']   = ! empty( $item->url )        ? $item->url        : '';
+		$atts['aria-current'] = ! empty( $item->current ) ? 'page' : '';
```

Comment:
@viktorix We need to add some spaces before the `=` to ensure the right indentations.

Reply:
I added the spacing [here](https://github.com/ClassicPress/ClassicPress/pull/1027).

Reply:
Indentation fix has been applied to this PR so closing PR 1022 now.